pocs/tests/conftest.py

```diff
@@ -41,3 +44,30 @@ def config_with_simulated_dome(config):
 @pytest.fixture
 def data_dir():
     return '{}/pocs/tests/data'.format(os.getenv('POCS'))
+
+
+def end_process(proc):
+    proc.send_signal(signal.SIGINT)
+    return_code = proc.wait()
```

Comment:
Did you want to return this `return_code`?

Reply:
Returning it would be a good idea from the point of view of making this a more generally useful function.  As it's currently used in these tests though I think any return value is just going to disappear down a black hole.  `end_process` is used as request finalizer for the `name_server` and `camera_server` fixtures, so only gets called when they go out of scope, at which point I don't think there's anything left to receive and act on the return value.

I should probably be checking the return value within `end_process` itself though, and emitting a warning if it's non-zero.  Might want to put a timeout on that wait, too, just in case.